src/components/timeline/Row.svelte

```diff
@@ -995,6 +1116,18 @@
     color: var(--st-utility-blue);
   }
 
+  :global(.row-external-event.st-button.icon svg) {
+    color: var(--st-gray-50);
+  }
+
+  :global(.row-external-event.st-button.icon:hover svg) {
+    color: var(--st-gray-70);
+  }
+
+  :global(.row-external-event.row-external-event-active.st-button.icon svg) {
+    color: var(--st-utility-blue);
+  }
+
```

Comment:
Unused css?

Reply:
Addressed in [remove unused CSS](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/19be56dc351cb30294eb00ac599eefd6d4840255).